packages/visx-demo/src/sandboxes/visx-xychart/Example.tsx

```diff
@@ -53,173 +52,171 @@ export default function Example({ height }: Props) {
         xAxisOrientation,
         yAxisOrientation,
       }) => (
-        <DataProvider theme={theme} xScale={config.x} yScale={config.y}>
-          <XYChart height={Math.min(400, height)}>
-            <CustomChartBackground />
-            <AnimatedGrid
-              key={`grid-${animationTrajectory}`} // force animate on update
-              rows={showGridRows}
-              columns={showGridColumns}
-              animationTrajectory={animationTrajectory}
-              numTicks={numTicks}
-            />
-            {renderBarStack && (
-              <AnimatedBarStack>
-                <AnimatedBarSeries
-                  dataKey="New York"
-                  data={data}
-                  xAccessor={accessors.x['New York']}
-                  yAccessor={accessors.y['New York']}
-                />
-                <AnimatedBarSeries
-                  dataKey="San Francisco"
-                  data={data}
-                  xAccessor={accessors.x['San Francisco']}
-                  yAccessor={accessors.y['San Francisco']}
-                />
-                <AnimatedBarSeries
-                  dataKey="Austin"
-                  data={data}
-                  xAccessor={accessors.x.Austin}
-                  yAccessor={accessors.y.Austin}
-                />
-              </AnimatedBarStack>
-            )}
-            {renderBarGroup && (
-              <AnimatedBarGroup>
-                <AnimatedBarSeries
-                  dataKey="New York"
-                  data={data}
-                  xAccessor={accessors.x['New York']}
-                  yAccessor={accessors.y['New York']}
-                />
-                <AnimatedBarSeries
-                  dataKey="San Francisco"
-                  data={data}
-                  xAccessor={accessors.x['San Francisco']}
-                  yAccessor={accessors.y['San Francisco']}
-                />
-                <AnimatedBarSeries
-                  dataKey="Austin"
-                  data={data}
-                  xAccessor={accessors.x.Austin}
-                  yAccessor={accessors.y.Austin}
-                />
-              </AnimatedBarGroup>
-            )}
-            {renderBarSeries && (
+        <XYChart theme={theme} xScale={config.x} yScale={config.y} height={Math.min(400, height)}>
+          <CustomChartBackground />
```

Comment:
💬  only `DataProvider` was removed, everything else in this file is a de-indentation